ocw_data_parser/static_html_generator.py

```diff
@@ -1,6 +1,6 @@
 import logging
-
-from .utils import is_json, get_correct_path, load_json_file
+import os
+from .utils import is_json, get_correct_path, get_page_from_url, load_json_file
```

Comment:
`get_page_from_url` doesn't exist in utils

Reply:
This is a merge artifact and has been corrected.